api/e2e/test/cluster/cluster.go

```diff
@@ -57,7 +57,7 @@ func GetModelClusterCredentials(clusterName string, vaultCfg config.VaultConfig)
 		Address: vaultCfg.Address,
 		Token:   vaultCfg.Token,
 	}
-	vaultClient, err := vault.NewVaultClient(vaultConfig)
+	vaultClient, err := vault.NewClient(vaultConfig)
```

Comment:
This change is made due to an update in the `gojek/mlp` version which feature new method names for the vault package.